src/multiplayer-servers/multiplayer-services/server-protection/protocols.md

```diff
@@ -0,0 +1,68 @@
+# Ports and Protocols
+
+Game server ports are often vulnerable to attacks. 
+GameFabric offers Protocols to protect your game server traffic with powerful SteelShield mitigations.
+
+## Pre-requisites
+
+In order to protect the your game server traffic, your GameFabric installation requires:
+
+- Protection support enabled, usually indicated by the presence of the Protection/SteelShield navigation item.
+- Sites that support SteelShield, e.g. bare-metal.
+
+## Mitigations
+
+SteelShield provides mitigations that are tailored to check and let through only specific game or network traffic.
```

Comment:
> game or network traffic

Weird to me since game traffic is network traffic.

```suggestion
SteelShield provides mitigations that are tailored to check and only allow specific network traffic.
```

Or if we want to make it clear that it could be for specific game traffic:

```suggestion
SteelShield provides mitigations that are tailored to check and only allow specific traffic, such as your game server's protocols.
```